ta/pkcs11/src/serializer.h

```diff
@@ -9,6 +9,9 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+struct pkcs11_client;
+struct pkcs11_session;
+
```

Comment:
```suggestion
#include "pkcs11_token.h" 
```
Why the declaration instead of an include?

Reply:
This prevents conflicts in the ordering of structures definition as here we don't care about the structures internals.

Maybe using `#include "pkcs11_token.h"` here is not an issue now but I fear later conflict issues when more feature will be implemented. This point is, including **serializer.h** does not mandated including **pkcs11_token.h**.